portfolio/src/main/webapp/javascript/map.js

```diff
@@ -0,0 +1,22 @@
+      var map;
```

Comment:
Maybe I missed it, but I don't see this file referenced anywhere - can it be deleted? (or maybe it needs to added somewhere)

Reply:
yeah this was my attempt to put the js into a separate file